strongbox-db-server/src/main/java/org/strongbox/db/server/EmbeddedJanusGraphWithCassandraServer.java

```diff
@@ -71,11 +74,10 @@ private CassandraDaemon provideCassandraInstance()
             return cassandraDaemon;
         }
 
-        System.setProperty("cassandra.config.loader",
-                           cassandraEmbeddedProperties.getCassandraConfigLoaderClassName());
+        System.setProperty("cassandra.config.loader", cassandraEmbeddedProperties.getCassandraConfigLoaderClassName());
 
         System.setProperty("cassandra-foreground", "true");
-        System.setProperty("cassandra.native.epoll.enabled", "false");
+        System.setProperty("cassandra.native.epoll.enabled", "true");
```

Comment:
FYI: This option can be left as `true` - Cassandra will ignore it on windows and systems which don't have proper netty-epoll support.